README.md

```diff
@@ -22,36 +36,235 @@ component written in native code. Using that component from DNX
+
+```sh
+PM> Install-Package Google.Logging.V2 -Prerelease
+```
+
+#### Example
+
+```c#
+using Google.Logging.V2;
+...
+
+Code coming soon
+```
+
+## Google Cloud Pub/Sub
+
+- [API Reference][cloud-pubsub-ref]
+- [API Overview][cloud-pubsub-docs]
+
+The Cloud Pub/Sub client library is in pre-release state and is more likely to have significant surface changes over time. Still, if you would like to experiment with it, we would welcome your feedback.
+
+#### Installation
+The package is available on Google's [public MyGet feed][google-myget-feed], where you'll also find the right Package Source to add in Visual Studio. Once done, the package can be installed in the following manner:
+
+```sh
+PM> Install-Package Google.Pubsub.V1 -Prerelease
+```
+
+#### Example
+
+```c#
+using Google.Pubsub.V1;
+...
+
+Code coming soon...
+```
+
+## Google Cloud Storage
+
+- [API Reference][cloud-storage-ref]
+- [API Overview][cloud-storage-docs]
+
+The Cloud Storage client library is in a more advanced state. However, there is no guarantee that the API surface will stay stable until we reach 1.0 - but we have high confidence that the libraries work and are usable.
+
+#### Installation
+The package is available on [NuGet](https://www.nuget.org/packages/Google.Storage.V1) and can be installed in the following manner:
+
+```sh
+PM> Install-Package Google.Storage.V1 -Prerelease
+```
+
+#### Example
+
+```c#
+using Google.Storage.V1;
+...
+
+var client = StorageClient.Create();
+
+// Create a bucket
+var bucketName = Guid.NewGuid().ToString(); // must be globally unique
+var bucket = client.CreateBucket(projectId, bucketName);
+
+// Upload some files
+var content = Encoding.UTF8.GetBytes("hello, world");
+var obj1 = client.UploadObject(bucketName, "file1.txt", "text/plain", new MemoryStream(content));
+var obj2 = client.UploadObject(bucketName, "folder1/file2.txt", "text/plain", new MemoryStream(content));
+
+// List objects
+foreach (var obj in client.ListObjects(bucketName, ""))
+{
+    Console.WriteLine(obj.Name);
+}
 
-- [C# API reference](http://googlecloudplatform.github.io/gcloud-dotnet/)
-- [Google Cloud Platform](https://cloud.google.com/docs/)
+// Download file
+using (var stream = File.OpenWrite("file1.txt"))
+{
+    client.DownloadObject(bucketName, "file1.txt", stream);
+}
+```
 
-Contributing
-------------
+## Contributing
 
 Contributions to this library are always welcome and highly encouraged.
 
 See [CONTRIBUTING] for more information on how to get started.
 
-Versioning
-----------
+## Versioning
 
 These libraries follow [Semantic Versioning](http://semver.org/).
 
 Anything with a major version of zero (``0.y.z``) should not be
 considered stable - anything may change at any time.
 
-License
--------
+## License
 
 Apache 2.0 - See [LICENSE] for more information.
 
 
 [CONTRIBUTING]:https://github.com/GoogleCloudPlatform/gcloud-dotnet/blob/master/CONTRIBUTING.md
```

Comment:
Absolute paths are **not** recommended.
https://help.github.com/articles/relative-links-in-readmes/